javascript/javascript3/redo/week2/fetch.js

```diff
@@ -0,0 +1,26 @@
+//3 steps below using promises and .then
+/////////////////////////////////////////////////
+new Promise(resolve =>
```

Comment:
I think this exercise was meant to build on the "Rewrite" part, where you rewrote setTimeout to be Promise-based. That way you could do this:
```js
  setTimeoutPromise(3000)
    .then(...)
```

And for the async, you could do:
```js
  await setTimeoutPromise(3000);
  const response = await fetch(...);
```

Your current solution just uses the callback-style version of setTimeout.

Reply:
No this part is only Fetching and waiting, not the rewrite or async.